pkgs/tools/networking/slack-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+# slack-cli must be configured using the SLACK_CLI_TOKEN environment
+# variable. Using `slack init` will not work because it tries to write
+# to the Nix store.
+
+{ stdenv, lib, fetchFromGitHub, makeWrapper, curl, jq }:
+
+stdenv.mkDerivation rec {
+  name = "slack-cli";
+  version = "0.18.0";
+
+  src = fetchFromGitHub {
+    owner = "rockymadden";
+    repo = "slack-cli";
+    rev = "v${version}";
+    sha256 = "022yr3cpfg0v7cxi62zzk08vp0l3w851qpfh6amyfgjiynnfyddl";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
This shouldn't be necessary. The standard build phase uses make, it does not need to be specified as a dependency.